src/main/scala/net/degoes/01-data.scala

```diff
@@ -119,10 +137,10 @@ object Data extends ZIOSpecDefault {
             test("simple") {
               final case class Address(street: String)
 
-              def extractStreet(address: Address): String = ???
+              def extractStreet(address: Address): String = address.street
```

Comment:
It works but the exercise say you should use pattern matching. 
I think it's to show you that you can do `case Address(street) => street`